metadata/metadata.go

```diff
@@ -10,47 +10,60 @@ import (
 const AppMetadataLabel = "io.buildpacks.lifecycle.metadata"
 
 type AppImageMetadata struct {
-	App        AppMetadata         `json:"app"`
-	Config     ConfigMetadata      `json:"config"`
-	Launcher   LauncherMetadata    `json:"launcher"`
-	Buildpacks []BuildpackMetadata `json:"buildpacks"`
-	RunImage   RunImageMetadata    `json:"runImage"`
-	Stack      StackMetadata       `json:"stack"`
+	App        AppMetadata         `json:"app" toml:"app"`
+	Config     ConfigMetadata      `json:"config" toml:"config"`
+	Launcher   LauncherMetadata    `json:"launcher" toml:"launcher"`
+	Buildpacks []BuildpackMetadata `json:"buildpacks" toml:"buildpacks"`
+	RunImage   RunImageMetadata    `json:"runImage" toml:"run-image"`
+	Stack      StackMetadata       `json:"stack" toml:"stack"`
 }
 
 type AppMetadata struct {
-	SHA string `json:"sha"`
+	SHA string `json:"sha" toml:"sha"`
+}
+
+type AnalyzedMetadata struct {
+	Repository string           `toml:"repository"`
```

Comment:
If we call this repository it should be just the repository instead of the entire tag reference. We should either break out the pieces of the reference or rename this.